miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/EditSampleController.java

```diff
@@ -981,18 +998,21 @@ protected void writeConfiguration(ObjectMapper mapper, ObjectNode config) {
       config.put(Config.DNASE_TREATABLE, targetSampleClass.getDNAseTreatable());
       config.putPOJO(Config.TARGET_SAMPLE_CLASS, Dtos.asDto(targetSampleClass));
       config.putPOJO(Config.SOURCE_SAMPLE_CLASS, Dtos.asDto(sourceSampleClass));
+      config.putPOJO(Config.BOX, newBox);
     }
   };
 
   private final class BulkCreateSampleBackend extends BulkCreateTableBackend<SampleDto> {
     private final SampleClass targetSampleClass;
     private final Project project;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this empty statement. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AEmptyStatementUsageCheck)